### PR TITLE
More difficult ch2/3 fights

### PR DESCRIPTION
**Pull request by bulian-bg**

### CODE REVIEW

File: ImprovedAnvil/scripts/areas/AR0410.baf [hunk 1]
```diff
@@ -157,7 +157,9 @@ END
 IF
   InMyArea(Player1)
   Global("Iacells","AR0410",1)
-  XPGT(Player1,1999999)
+  OR(2)
+    XPGT(Player1,1999999)
+    Global("Iahardmode","Global",1)
```
Comment (critto-bg) [inline ImprovedAnvil/scripts/areas/AR0410.baf]: It's GLOBAL, uppercase

**Review by critto-bg — CHANGES_REQUESTED** [review]

File: ImprovedAnvil/scripts/areas/AR0410.baf (hunk 1 above)
Comment (bulian-bg) [reply ImprovedAnvil/scripts/areas/AR0410.baf]: well that was annoying

Comment (critto-bg) [reply ImprovedAnvil/scripts/areas/AR0410.baf]: You should at least do a compile test when you make changes, compilers are there for a reason - to catch these things :-)

**Review by critto-bg — APPROVED** [review]
So this is only ch2/3, right? I'll re-check these later and add an optional component. Right now the whole installer is dismantled in my local copy because I'm implementing the custom kits component.

**Comment by bulian-bg** [comment]
yes once the PC has 2,350,000 XP the patch ceases to exist.  really once the party gets through the first 1-2 major quest lines, the patch will have less and less impact.

**Comment by critto-bg** [comment]
ar0511 has some difficulty checks as well that you've missed (the bridge inn, second floor)

**Comment by critto-bg** [comment]
> ar0511 has some difficulty checks as well that you've missed (the bridge inn, second floor)

Added this check when merging.

**Comment by critto-bg** [comment]
Merged.

**Comment by bulian-bg** [comment]
> > ar0511 has some difficulty checks as well that you've missed (the bridge inn, second floor)
> 
> Added this check when merging.

great catch!

**Pull request closed without merging**